app/src/main/java/jetzt/machbarschaft/android/view/splash/SplashPresenter.kt

```diff
@@ -104,8 +130,93 @@ class SplashPresenter: SplashContract.Presenter {
     private fun onRequestRegistrationDataFailure(throwable: Throwable) {
         throwable.printStackTrace()
         view?.hideLoadingDialog()
-
         view?.showError()
     }
 
+
+    /**
+     * callback if the registration request is successful
+     */
+    private fun onRequestRegistrationDataSuccessful(response: CreateUserResponse) {
+        println("successful")
```

Comment:
`println` prints to stdout, not to the Android log,

- [ ] Use `Log.d`, `Log.i`, ... instead